installation/noobs.md

```diff
@@ -1,50 +1,20 @@
 # NOOBS
 
-**New Out Of Box Software (NOOBS)** is an easy operating system installation manager for the Raspberry Pi.
+**Note: We now recommend that you use [Raspberry Pi Imager](https://www.raspberrypi.org/software/) to install an operating system, instead of using NOOBS. If you are buying an SD card with software pre-installed on it, then we do still recommend buying one with NOOBS on it.**
```

Comment:
The "If you are buying ... then we do still recommend ..." text feels a little bit awkward. Perhaps this paragraph should just say something like:

**Note: We now recommend that you use [Raspberry Pi Imager](https://www.raspberrypi.org/software/) to install an operating system onto a blank SD card, instead of using NOOBS.**

Reply:
Gordon asked that the page mention NOOBS is still recommended for pre-installed SD cards.

Reply:
s/recommended/available/ ? :wink: 

**Note: We now recommend that you use Raspberry Pi Imager to install an operating system onto a blank SD card, instead of using NOOBS. Alternatively, you can buy an SD card with NOOBS pre-installed.**

:grey_question:

Reply:
I thought about that, but I think it is probably better to still say NOOBS is recommended for pre-installed cards. The reason for this is that otherwise it sounds like we are saying that you can use NOOBS on a pre-installed card, but that it is not recommended, which is not the case.

Reply:
Fair enough, perhaps simply removing the "do still" from your original text resolves the "awkward feeling" I had :slightly_smiling_face:

Reply:
Fixed.